scaleway/resource_container_namespace_test.go

```diff
@@ -147,6 +148,34 @@ func TestAccScalewayContainerNamespace_Basic(t *testing.T) {
 	})
 }
 
+func TestAccScalewayContainerNamespace_DestroyRegistry(t *testing.T) {
+	tt := NewTestTools(t)
+	defer tt.Cleanup()
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: tt.ProviderFactories,
+		CheckDestroy: resource.ComposeTestCheckFunc(
+			testAccCheckScalewayContainerNamespaceDestroy(tt),
+			testAccCheckScalewayContainerRegistryDestroy(tt),
+		),
+		Steps: []resource.TestStep{
+			{
+				Config: `
+					resource scaleway_container_namespace main {
+						name = "test-cr-ns-01"
+						destroy_registry = true
+					}
+				`,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckScalewayContainerNamespaceExists(tt, "scaleway_container_namespace.main"),
```

Comment:
Can you add a test to verify the registry is indeed gone?

Reply:
This is tested [here](https://github.com/scaleway/terraform-provider-scaleway/pull/1321/files#diff-931a1d8610baac01c7be3042178a5e58140873a2c93308f1d5d858127a65f0feR160)